README.md

```diff
@@ -148,7 +148,21 @@ Interpretation:
 
 The Steps Stats are based on the cards from the selected deck or collection that were first reviewed in the last 1 month/year or deck life and have at least two reviews.
 
-These stats are helpful for fine-tuning your (re)learning steps to achieve your desired retention in the short-term reviews. To learn more about learning steps, please read [learning steps](https://docs.ankiweb.net/deck-options.html#learning-steps) in the  Anki manual.
+These stats are helpful for fine-tuning your (re)learning steps to achieve your desired retention in the short-term reviews. 
+
+How your learning steps affect the intervals:
+- If you firstly rate again, the interval is the 1st learning step.
+- If you firstly rate again and then rate good in a new card, the interval is the 2nd learning step.
+- If you firstly rate good, the interval is the 2nd learning step.
+- If you firstly rate hard, the interval is (1st learning step + 2nd learning step) / 2.
+
+So the 1st recommended learning step is based on the stability of your cards where you rate again in the first review. 
```

Comment:
So the 1st recommended learning step is based on the stability of your cards where you press Again during the first review.